Resources/Prototypes/Roles/Jobs/Medical/chief_medical_officer.yml

```
@@ -45,11 +45,11 @@
     components:
     - type: PsionicBonusChance #Nyano - Summary: makes it more likely to become psionic.
       flatBonus: 0.025
-  # Shitmed Change, DeltaV role modifiers bad
-  # - !type:AddComponentSpecial
-  #   components:
-  #   - type: SurgerySpeedModifier
-  #     speedModifier: 2.5 - end DeltaV
+  # Shitmed Change
+  #- !type:AddComponentSpecial
+  #  components:
+  #  - type: SurgerySpeedModifier
+  #    speedModifier: 2.5
```

As stated on discord, this is wrong, got it's DeltaV comment nuked and the spaces got removed where they weren't needed.
```suggestion
   # Shitmed Change. DeltaV - role modifiers bad
   #- !type:AddComponentSpecial
   #  components:
   #  - type: SurgerySpeedModifier
   #    speedModifier: 2.5
```